docs/privacyidea.md

```diff
@@ -216,6 +216,41 @@ Use the following example:
         'excludeClientIPs'  => array("10.0.0.0-10.2.0.0", "192.168.178.2"),
     ),
 
+    /**
+     *  This filter is optional, if you want to disable clients by entityID, you should enable it.
```

Comment:
Since this si currently the only documentation, could you please add a bit more?

I understand that not a client is disabled, but the 2FA for certain Service Providers, which are identified by an entityID is disabled?

What happens if the first authentication request originiates from such an excluded Service Provider? Then the user will only have to provide his password in the authsources?

And if then the user visits an SP, that is not excluded? Will the user have to re-authenticate?

Maybe you could also add this information here in the docs.

Reply:
It is exacly as you say. I will add a bit more details.